zboss/doc/zboss_configuration.rst

```diff
@@ -11,7 +11,6 @@ The ZBOSS stack is distributed as a set of precompiled linkable libraries.
 These libraries are available in the following versions:
 
 * `ZBOSS production libraries`_
-* `ZBOSS development libraries`_
 
 Each version supports different Zigbee device roles, with each variant having its own configuration.
 This allows you to scale the application and select the most suitable set of features.
```

Comment:
I feel like these lines don't need to be here now that we only offer one set of libraries.

Reply:
But within the set of libraries there are still variants.